src/tsutil/ts_diags.cc

```diff
@@ -25,7 +25,13 @@
 #include "tsutil/ts_diag_levels.h"
 #include "tsutil/ts_errata.h"
 
-static const bool INITIALIZED = []() -> bool {
+// It seems the linker will omit this unit and the INITIALIZED lambda if it detects
+// that no other function is being called from this unit. So the following function
+// it's just a ugly and dirty way to force the linker and compiler to include this
+// and actually invoke the lambda down below.
+void Do_Not_Omit_Errata_Initialization() {};
```

Comment:
I think it would a avoid unnecessary complexity to just have a function to do initialization for Errata, rather than doing the initialization in the dummy INITIALIZED variable.  I don't think we can be confident that the initialization of SEVERITY_NAMES within libswoc is done with constant initialization.  So, it could happen *after* the initialization of INITIALIZED.

Reply:
Exactly, this should happen early on, and the best way is using the `INITIALIZED`, I did try calling a function to do this, but I did it probably too late in the flow. I can try just right after we start the `main` in `traffic_server.cc`

Reply:
But, if the three variables are themselves used during non-local initialization, there is no guarantee (due to Initialization Fiasco) that they will be initialized before they are used.  Or, as I said, that the dynamic initialization of SEVERITY_NAMES will happen before INITIALIZED is initialized.